README.md

```diff
@@ -78,3 +91,11 @@ Feel free to check the [issues page](../../issues/).
 Give a ⭐️ if you like this project!
 
 
+## Huge Thanks
+
+I give my huge thanks for [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio). For wonderful template on [Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)).  
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL] Please add a valid license link here because that one you gave is broken.
- [OPTIONAL] Also, I have to use this place to request one more change. On line 33, you gave the wrong link to clone. Please update it.
![Screenshot (248)](https://user-images.githubusercontent.com/67087664/164456237-c73652f2-01b6-491e-acbe-73068ca05261.png)

Reply:
I did solve it